### PR TITLE
Fix broken links across posts

### PR DESCRIPTION
**Pull request by oskarth**
Closes https://github.com/vacp2p/vacp2p.github.io/issues/36

Please have a look if I missed or botched anything.

@serhanwbahar once you accepted invite you should be able to review

### CODE REVIEW

**Review by jm-clius — APPROVED** [review]
LGTM. Thanks!

**Review by serhanwbahar — APPROVED** [review]

File: _posts/2019-12-03-fixing-whisper-with-waku.md [hunk 1]
```diff
@@ -287,8 +287,8 @@ iterative work with results on the order of weeks.
 
 ### Progress so far
 
-In short, we have a [Waku version 0 spec up](https://specs.vac.dev/specs/waku.html) as well as a [PoC](https://github.com/status-im/nim-eth/pull/120) for backwards compatibility. In the coming weeks, we are going to solidify the specs, get a more fully featured PoC for [Waku mode](https://github.com/status-im/nim-eth/pull/114). See [rough roadmap](https://github.com/vacp2p/pm/issues/5), [project board](https://github.com/orgs/vacp2p/projects/2) and progress thread on the [Vac forum](https://forum.vac.dev/t/waku-project-and-progress/24).
+In short, we have a [Waku version 0 spec up](https://rfc.vac.dev/spec/5) as well as a [PoC](https://github.com/status-im/nim-eth/pull/120) for backwards compatibility. In the coming weeks, we are going to solidify the specs, get a more fully featured PoC for [Waku mode](https://github.com/status-im/nim-eth/pull/114). See [rough roadmap](https://github.com/vacp2p/pm/issues/5), project board [link deprecated] and progress thread on the [Vac forum](https://forum.vac.dev/t/waku-project-and-progress/24).
```
Comment (staheri14) [inline _posts/2019-12-03-fixing-whisper-with-waku.md:290]:
```suggestion
In short, we have a [Waku version 0 spec up](https://rfc.vac.dev/spec/5) as well as a [PoC](https://github.com/status-im/nim-eth/pull/120) for backwards compatibility. In the coming weeks, we are going to solidify the specs, get a more fully featured PoC for [Waku mode](https://github.com/status-im/nim-eth/pull/114). See [rough roadmap](https://github.com/vacp2p/pm/issues/5), [project board](link deprecated) and progress thread on the [Vac forum](https://forum.vac.dev/t/waku-project-and-progress/24).
```

Do you want `link deprecated` to be visible?

**Review by staheri14 — APPROVED** [review]
Thanks!

File: _posts/2019-12-03-fixing-whisper-with-waku.md (hunk 1 above)
Comment (oskarth) [reply _posts/2019-12-03-fixing-whisper-with-waku.md:290]: yeah

**Pull request merged**